test/lib/alloc-ringpop.js

```diff
@@ -64,8 +64,8 @@ function allocRingpop(name, options) {
 }
 
 function semiRandPort() {
-    var base = 20000;
-    var rand = Math.floor(Math.random() * 20000);
+    var base = 10000;
```

Comment:
Tests were flaky on linux by picking a random port in the ephemeral range.

By picking a random port in the non-ephemeral range the tests no longer fail with EADDRINUSE